CHANGELOG.rst

```diff
@@ -15,6 +15,11 @@ in development
 * Fix for `data` is dropped if `message` is not present in notification. (bug-fix)
 * Remove now deprecated Fabric based remote runner and corresponding
   ``ssh_runner.use_paramiko_ssh_runner`` config option. (cleanup)
+* Remove ``sudo`` support for local runner actions. This also means removing ``sudo`` parameter
+  from ``core.local`` action and removing ``core.local_sudo`` action completely.
+* Allow operator to disable local runner actions by setting ``local_runner.enable`` config option
```

Comment:
Yeah, that's a good point - we do actually have `enabled` attribute on the `RunnerTypeDB` model, but it gets overwritten when running register content when runners are registered.

I will look into it and see if I can make this existing attribute work without overwriting it on runner registration. I would definitely prefer to get this to work over re-inventing a new, not so consistent approach for disabling a runner.